### PR TITLE
sc - create personal schedules dropdown component

### LINKED ISSUE #6: Create a dropdown component for selecting a personal schedule
Later on, we want users to be able to add a course to any personal schedule. For that we, need a nice dropdown component that lists all of the current personal schedules.

Using frontend/src/main/components/Quarters/SingleQuarterDropdown.js as a model, create a component that takes a list of personal schedule objects as a parameter (i.e. a prop), and produces a dropdown that has a list of personal schedules.

Each schedule in the dropdown should have as it's value the id of the personal schedule, and as the option that displays, the quarter (in QXX format, e.g. F22) followed by the first 15 characters of the name of the personal schedule.

Note that another issue is making sure that the names will not be longer than 15 characters in the future, but you don't need to wait for that to be done. For now, just display the first 15 characters of whatever you get.

### PR DESCRIPTION
**Pull request by sophiacheng01**
In this PR, we add the code to create a dropdown component along with tests using fixtures. 

To run my code and visualize the component clone my repo, cd into frontend, and run npm storybook.

The storybook link on the docs-qa repo is the following:
https://ucsb-cs156-f22.github.io/f22-5pm-courses-docs-qa/storybook-qa/sophia-dropdown/?path=/docs/components-personalschedules-personalscheduledropdown--one-personal-schedule

Here is the visualization through storybook before you click on the dropdown for three schedules and the following picture is after you click:
Screen Shot 2022-11-27 at 11 47 42 AM

Screen Shot 2022-11-27 at 11 47 51 AM

Closes #6

### CODE REVIEW

**Review by hauntedghosts — DISMISSED** [review]
Tested locally and on QA and produced the same result, LGTM

File: frontend/src/fixtures/personalScheduleFixtures.js [hunk 1]
```diff
@@ -54,11 +77,11 @@ const personalScheduleFixtures = {
             "hostedDomain": "ucsb.edu",
             "admin": true
           },
-          "description": "DATA STRUCT ALGOR",
-          "quarter": "20221",
-          "name": "CS130A"
+          "description": "My Plan for Fall",
+          "quarter": "20229",
```
Comment (pconrad) [inline frontend/src/fixtures/personalScheduleFixtures.js:81]: 20229 isn't a valid quarter... the last digit should be 1 for Winter, 2 for Spring, 3 for Summer, and 4 for Fall.   I think you probably want to change this to 20224

**Review by pconrad — CHANGES_REQUESTED** [review]
One small issue in the fixtures.. otherwise, lgtm

File: frontend/src/fixtures/personalScheduleFixtures.js (hunk 1 above)
Comment (pconrad) [inline frontend/src/fixtures/personalScheduleFixtures.js:81]:
```suggestion
          "quarter": "20224",
```

**Review by pconrad — CHANGES_REQUESTED** [review]

**Review by pconrad — APPROVED** [review]
LGTM!

**Pull request merged**